docs/how-to/data-plane-installation.md

```diff
@@ -0,0 +1,74 @@
+# Installing Kuadrant data-plane into an existing OCM Managed Cluster
+
+## Introduction
+This walkthrough will show you how to install and setup the Kuadrant Operator into an [OCM](https://open-cluster-management.io/) [Managed Cluster](https://open-cluster-management.io/concepts/managedcluster/).
+
+Steps:
+- OCM should be installed and you should have an existing OCM hub cluster (we should cover the specifics of that setup in the cp install guide which this should link to) with the control plane components installed (IE follow the CP guide first)
+- Install Istio version x.y as this is the gateway provider and we do not install this (we can provide a ref set of install steps)
+- Install OLM as the service protection components are delivered by OLM currently
+- Ensure this cluster is added as a managed cluster to your OCM hub (can be same cluster or another)
+- create the ManagedClusterAddon resource in the managed cluster namespace to trigger the install of kuadrant into the spoke cluster
+
+## Prerequisites
+* Access to an OCM Managed Cluster, which has already been bootstrapped and registered with a hub cluster
```

Comment:
we should specify the version of OCM to use >=0.6.0

These versions are captured in the dependencies.make file

Reply:
have a dropped a mention of this guide into the pre-requisites now